src/languages/idl.js

```diff
@@ -0,0 +1,58 @@
+/*
+Language: OMG IDL
```

Comment:
I think I should attempt the second option. However, some issues may arise from merging OMG IDL and WebIDL grammars. For instance, WebIDL keywords shouldn't be highlighted for a snippet using OMG IDL.